packages/frontend/src/components/Buttons/AccountButton.tsx

```diff
@@ -1,14 +1,27 @@
 import { shortenAddress, useEthers } from '@usedapp/core'
+import { useCallback } from 'react'
+import { removeWalletLinkStorage } from 'src/utils/removeWalletLinkStorage'
 import styled from 'styled-components'
 
 import { Button } from './Button'
 import { ConnectWalletButton } from './ConnectWalletButton'
 
 export const AccountButton = () => {
-  const { account } = useEthers()
+  const { account, deactivate } = useEthers()
+
+  const disconnect = useCallback(async () => {
+    localStorage.removeItem('walletconnect')
+    removeWalletLinkStorage()
+    deactivate()
+  }, [deactivate])
 
   return account ? (
-    <ConnectedButton view="secondary">{shortenAddress(account)}</ConnectedButton>
+    <>
+      <ConnectedButton view="secondary" onClick={disconnect}>
+        Disconnect
+      </ConnectedButton>
+      <ConnectedButton view="secondary">{shortenAddress(account)}</ConnectedButton>
+    </>
   ) : (
     <ConnectWalletButton view="secondary" />
```

Comment:
Same problem with Web3Modal as described here: https://github.com/TrueFiEng/devcon-raffle/pull/139#discussion_r871210394. Maybe @mrushkova you could solve this here?

Reply:
Yes, [solved](https://github.com/TrueFiEng/devcon-raffle/pull/139/commits/e0a7e0a4e7bdaab13c3a7366c43670e020a6b30e).